test/cypress/components/filterable-list-control.js

```diff
@@ -0,0 +1,75 @@
+export class FilterableListControl {
+
+  filterableListControl() {
+    return cy.get( '.o-filterable-list-controls' );
+  }
+
+  filterableElement( selector ) {
+    return this.filterableListControl().get( selector );
+  }
+
+  filterableListElement( name ) {
+    return cy.get( `#o-filterable-list-controls_${ name }` );
+  }
+
+  openFilterableListControl() {
+    return cy.get( '#o-filterable-list-controls' )
+      .find( 'button' ).first().click( { force: true } );
+  }
```

Comment:
Can we get rid of this and replace uses of it with `showFilter()`?

Reply:
Yes I removed `openFilterableListControl` and replaced with `showFilter`